botfront/imports/api/graphql/story/schemas/stories.types.graphql

```diff
@@ -0,0 +1,26 @@
+type Query {
+    stories(projectId: String! language: String! queryString: String!): [Story]
+}
+
+type Story {
+    _id: String
+    story: String
+    title: String
+    storyGroupId: String
+    projectId: String
+    events: [String]
+    branches: [Branch]
```

Comment:
why not [Story]

Reply:
for which line, branches: [branch]? I separated them since the branch schema doesn't have several fields that are in a root level story. If you think It makes sense to reuse the story instead I'm ok with it.